src/arrayMethodSort.js

```diff
@@ -1,11 +1,38 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const defaultFunction = (a, b) => {
+    const strA = String(a);
+    const strB = String(b);
+
+    if (strA > strB) {
+      return 1;
+    } else if (strA === strB) {
+      return 0;
+    }
```

Comment:
Else statement is redudant
```suggestion
    if (strA > strB) {
      return 1;
    } 
    
    if (strA === strB) {
      return 0;
    }
```